src/Service.Tests/SqlTests/GraphQLMutationTests/MsSqlGraphQLMutationTests.cs

```diff
@@ -58,6 +58,29 @@ public async Task InsertMutation()
             await InsertMutation(msSqlQuery);
         }
 
+        /// <summary>
+        /// <code>Do: </code> Inserts new Publisher with name = 'New publisher'
+        /// <code>Check: </code> Mutation fails because the database policy is not satisfied by the mutation.
```

Comment:
`the database policy is not satisfied by the mutation.` This is self-documented by the test name. It would be helpful to know **why** the database policy is not satisfied.